### PR TITLE
Feature/fe 712 invite friends

### PR DESCRIPTION
**Pull request by yevt0**
https://mindojo.atlassian.net/browse/FE-712

### CODE REVIEW

File: src/js/main.coffee [hunk 1]
```diff
@@ -112,12 +128,13 @@ initCourseSelector = ->
     return false
   )
 
-
```
Comment (nikdojo) [inline src/js/main.coffee]: @yevt0 Let's move the code into separate module?

Comment (yevt0) [inline src/js/main.coffee]: do you mean initCourseSelector ?

Comment (nikdojo) [inline src/js/main.coffee]: I think so... first time see this task

Comment (nikdojo) [inline src/js/main.coffee]: It's just copy-paste, no need for a separate task imho

Comment (yevt0) [inline src/js/main.coffee]: It won't work, there are lots of global functions

Comment (nikdojo) [inline src/js/main.coffee]: So? I don't understand what's the problem. `main.coffee` is totaly the same module as others, it's just separate files with `require()`.

Up2u.

**Pull request merged**